src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -567,7 +567,6 @@
     <Reference Include="System.Reflection.Emit.Lightweight" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\System.Threading.Tasks.Extensions\src\System.Threading.Tasks.Extensions.csproj" />
```

Comment:
this was causing problems when building locally. Full build works but incremental build was complaining that implementation of ValueTask was different that ref (some dummy CCI attribute was causing some issues).

I haven't seen any issues after removing this but I had to do full rebuild